tgui/packages/tgui/interfaces/PreferencesMenu/QuirksPage.tsx

```diff
@@ -300,6 +375,13 @@ export const QuirksPage = (props, context) => {
 
                       act('remove_quirk', { quirk: quirk.name });
                     }}
+                    onCustomizeClick={(e: Event, quirkName, quirk) => {
+                      e.stopPropagation();
+
+                      quirk.customization_expanded = true;
```

Comment:
No! Don't mutate the data you get, that is illegal

Reply:
Set local state instead

Reply:
worth mentioning i abandoned this approach in favor of the collapsable component, but if uselocalstate is how we store variables within windows and shit i will absolutely switch to that